tests/installation/scc_registration.pm

```diff
@@ -31,9 +31,9 @@ sub run {
 
 sub post_fail_hook {
     my ($self) = @_;
-    $self->SUPER::post_fail_hook;
     verify_scc;
     investigate_log_empty_license;
+    $self->SUPER::post_fail_hook;  
```

Comment:
As being said, I would appreciate if we could understand why it doesn't work instead of postponing our work.

Reply:
This change doesn't help, only works in my local. I will close this PR and we can discuss what to do next. I will leave comments on https://progress.opensuse.org/issues/48128.

Reply:
Sure, no stress.